core/src/components/breadcrumbs/test/basic/breadcrumbs.e2e.ts

```diff
@@ -2,11 +2,24 @@ import { expect } from '@playwright/test';
 import { test } from '@utils/test/playwright';
 
 test.describe('breadcrumbs: basic', () => {
-  test('should not have visual regressions', async ({ page }) => {
-    await page.goto(`/src/components/breadcrumbs/test/basic`);
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/src/components/breadcrumbs/test/basic');
+  });
 
+  test('should not have visual regressions', async ({ page }) => {
     await page.setIonViewport();
 
     await expect(page).toHaveScreenshot(`breadcrumb-diff-${page.getSnapshotSettings()}.png`);
   });
+
+  test('should have color attribute', async ({ page }) => {
+    test.info().annotations.push({
+      type: 'issue',
+      description: 'https://github.com/ionic-team/ionic-framework/issues/25446',
+    });
+
+    const breadcrumbs = page.locator('#color');
```

Comment:
Is it possible to use a unit test here instead of an e2e test?

It looks like we only need to check the output in the DOM, and unit tests run much faster than e2e tests.

You could do something like this:

```js
it('should have color attribute', async () => {
  const page = await newSpecPage({
    components: [Breadcrumb, Breadcrumbs],
    html: `
      <ion-breadcrumbs color="danger">
        <ion-breadcrumb>First</ion-breadcrumb>
      </ion-breadcrumbs>
    `
  });
  const breadcrumbs = page.body.querySelector('ion-breadcrumbs');
  expect(breadcrumbs.hasAttribute('color')).toBe(true);
});
```

We use Stencil's unit testing library: https://stenciljs.com/docs/unit-testing